interface/forms/fee_sheet/new.php

```diff
@@ -11,13 +11,15 @@
  * @copyright Copyright (c) 2018 Brady Miller <brady.g.miller@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
-use OpenEMR\Core\Header;
 
 require_once("../../globals.php");
 require_once("$srcdir/FeeSheetHtml.class.php");
 require_once("codes.php");
 require_once("$srcdir/options.inc.php");
 
+use OpenEMR\Core\Header;
+use OpenEMR\Billing\Billing;
```

Comment:
Should keep the use ordered alphabetically so easier in future when we have more of them.

Reply:
ok, that's wise